moon/cfc_chat_transit/server/init.moon

```diff
@@ -76,4 +77,39 @@ ChatTransit.ReceiveMessage = (ply, text, teamChat) =>
 
     @Logger\debug "Sent message '#{text}' to websocket"
 
+ChatTransit.PlayerConnected = (ply) =>
+
+    steamName = ply\Nick!
+    steamId = ply\SteamID64!
+    irisId = "none"
+
+    struct =
+        Realm: Realm
+        Type: "connect"
+        SteamName: steamName
+        SteamId: steamId
+        IrisId: irisId
+
+    message = TableToJSON struct
+
+    @WebSocket\write message
+
+ChatTransit.PlayerDisconnected = (ply) =>
+    steamName = ply\Nick!
+    steamId = ply\SteamID64!
+    irisId = "none"
+
+    struct =
+        Realm: Realm
+        Type: "disconnect"
+        SteamName: steamName
+        SteamId: steamId
+        IrisId: irisId
+
+    message = TableToJSON struct
+
+    @WebSocket\write message
+
 hook.Add "PlayerSay", "CFC_ChatTransit_MessageListener", ChatTransit\ReceiveMessage, HOOK_MONITOR_LOW
+hook.Add "PlayerConnect", "CFC_ChatTransit_ConnectListener", ChatTransit\PlayerConnected
```

Comment:
`PlayerConnect` won't give you what you're looking for. It only passes `name` and `ip`

You'll probably want to use the `player_connect` game event: https://wiki.facepunch.com/gmod/Game_Events